convert_to_multi_instance.sh

```diff
@@ -315,6 +318,21 @@ function check_CSCR() {
 
 }
 
+function removeNSS(){
+    ${OC} get nss --all-namespaces | grep nss-managedby-odlm | while read -r line; do
+        local namespace=$(echo $line | awk '{print $1}')
+        info "patching and deleting namespace scope nss-managedby-odlm in namespace $namespace"
+        ${OC} patch nss nss-managedby-odlm -n ${namespace} --type=merge -p '{"metadata": {"finalizers":null}}'
+        ${OC} delete nss nss-managedby-odlm -n ${namespace} || error "unable to delete namespace scope nss-managedby-odlm in ${namespace}"
```

Comment:
Hi @bluzarraga 

The place where we delete NSS CR is great!
Those NSS CR's will be created and updated by ODLM and cs-operator. It is good that we call `removeNSS` after scaling down cs-operator and ODLM.

I am also wondering that if we could try to remove NSS CR's first?
If the normal deletion is timeout/not working, we could raise a warning msg and patch the finalizer to `null` for that CR.
The main consideration is that
1. We did not touch namespace-scope operator in the script, it should be able to remove NSS CR correctly.
2. When namespace-scope operator is deleting the NSS CR, it will delete all RBAC's related to this CR, which are used to extend permission to other namespace. If we leave those RBAC in the cluster, it still grants operators the access to unexpected namespace. 

Reference on [waiting for CR to be deleted](https://github.com/IBM/ibm-common-service-operator/blob/95843d0f3c48f669c494a0f6ea4e79dc5a5e2af5/force-uninstall.sh#L324)

Reply:
If it times out deleting, how can we determine which RBAC resources need to be deleted? We might need a separate PR to recover from this kind of situation, because it will be very difficult to troubleshoot if after conversion, the NSS CRs say one thing, but in actuality the RBAC resources are different

Reply:
I think the deletion is time out only due to one situation where NSS controller is not able to remove finalizer in NSS CR.

And the only reason why NSS controller could not remove the finalizer is that the controller is gone(NSS operator not working properly)

Given the case that NSS operator should stay in the cluster after conversion, we might never patch the finalizer and always have NSS operator to handle the deletion.

If the deletion is truly time out in conversion, should we stop the script and ask user to manually troubleshooting the NSS operator first?

> how can we determine which RBAC resources need to be deleted?

If we would like to manually clean RBAC resources, we could [list and delete all RBAC with certain label in targetNamespace](https://github.com/IBM/ibm-namespace-scope-operator/blob/30e66e340420ef5b816a70c432bf6330fc2904ad/controllers/namespacescope_controller.go#L349)